docs/tutorials/deploy-vm/iso.md

```diff
@@ -0,0 +1,165 @@
+# Deploy a VM with ISO
+
+This tutorial describes the end-to-end workflow for importing an ISO image from a URL, deploying a VM from the ISO image, and installing the guest OS via VM's web console.
+
+## Import ISO from URL
+
+ISO images are usually imported by an infrastructure administrator into a Content Library associated with a namespace or cluster. To check if any ISO images are available in a namespace or cluster, use the following commands:
+
+```shell
+$ kubectl get -n <NAMESPACE> vmi -l image.vmoperator.vmware.com/type=ISO
+
+$ kubectl get cvmi -l image.vmoperator.vmware.com/type=ISO
+```
+
+DevOps can also import OVF/ISO images from a URL using the image registry operator's [ContentLibraryItemImportRequest](https://github.com/vmware-tanzu/image-registry-operator-api/blob/main/api/v1alpha1/contentlibraryitemimportrequest_types.go) API. The following example demonstrates how to import an upstream Rocky Linux ISO image:
```

Comment:
`s/DevOps/A DevOps user`

Reply:
Also, `using the image registry API` is fine. We only call ours the VM Op API because of legacy reasons. It should likely be called the VM API.

Reply:
Finally, `how to import a Rocky Linux ISO from a remote URL.`

Reply:
Addressed all, thanks for the suggestions.